docs/user/recipes/multipart-mixed.rst

```diff
@@ -0,0 +1,96 @@
+.. _nested-multipart-forms:
+
+Parsing Nested Multipart Forms
+==============================
+
+Out of the box, Falcon does not offer official support for parsing nested
+multipart forms (i.e., where multiple files for a single field are transmitted
+using a nested ``multipart/mixed`` part).
+
+.. note::
+    Nested multipart forms are considered deprecated according to the
+    `living HTML5 standard
+    <https://html.spec.whatwg.org/multipage/form-control-infrastructure.html>`_
+    and
+    `RFC 7578, Section 4.3 <https://tools.ietf.org/html/rfc7578#section-4.3>`_.
+
+However, if your app needs that, nested forms can be handled in the same
+fashion as any other part embedded in the form -- by installing an appropriate
+media handler.
+
+Let us extend the multipart form parser :attr:`media handlers
+<falcon.media.multipart.MultipartParseOptions.media_handlers>` to recursively
+parse embedded forms of the ``multipart/mixed`` content type:
+
+.. code:: python
+
+    import falcon
+    import falcon.media
+
+    parser = falcon.media.MultipartFormHandler()
+    parser.parse_options.media_handlers['multipart/mixed'] = (
+        falcon.media.MultipartFormHandler())
+
+.. note::
+    Here we create a new parser (with default options) for nested parts,
+    effectively disallowing further recursion.
+
+    If traversing into even deeper multipart form hierarchy was desired, we
+    could just reuse the same parser.
+
+Let us now use the nested form enabled parser in an app:
```

Comment:
Using three modifiers for "parser" makes the sentence a little difficult to read. Perhaps reword to "nesting-aware parser" or similar?